api/token.go

```diff
@@ -535,10 +548,18 @@ func (a *API) setCookieToken(config *conf.Configuration, tokenString string, ses
 	return nil
 }
 
-func (a *API) clearCookieToken(ctx context.Context, w http.ResponseWriter) {
-	config := getConfig(ctx)
+func (a *API) clearCookieTokens(config *conf.Configuration, w http.ResponseWriter) {
+	a.clearCookieToken(config, "access_token", w)
+	a.clearCookieToken(config, "refresh_token", w)
+}
+
+func (a *API) clearCookieToken(config *conf.Configuration, name string, w http.ResponseWriter) {
+	cookieName := config.Cookie.Key
+	if name != "" {
+		cookieName += "_" + name
```

Comment:
Using `_` as the delimiter might in the future make it hard to parse given `supabase_access_token` the suffix also has `_` so you have to split and pop off the first vs just split, but not a big deal. hold be fine unless one wants a more clear delimiter. 

> A <cookie-name> can contain any US-ASCII characters except for: the control character, space, or a tab. It also must not contain a separator characters like the following: ( ) < > @ , ; : \ " / [ ] ? = { }.

Reply:
hmm that's true but i can't think of a use case where it will be require to parse the cookie name? anyway, i can rename it to key_accesstoken & key_refreshtoken

Reply:
I have a preference for `sb:access_token` and `sb:refresh_token`, wdyt?

Reply:
i think the cookie name can't contain the ":" separator (https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#attributes)

Reply:
Hm, strange, I'm using `sb:token` in gotrue-js and it works fine, and IIRC it was because auth0 was doing that. Then can de do `sb.access_token` instead?

Reply:
I also see cookie name examples used `-`.

Perhaps:

`sb-access-token`

After some thought, the parsing the delimiter isn't all that critical.

And the `-` follows conventions like `x-use-cookie`